packages/dear-imgui/cpp/node/README.md

```
@@ -11,4 +11,8 @@ Open a developer terminal then run the following:
 - `cd ..\..\node`
 - `msbuild /p:Platform=x64 /p:Configuration=Release /p:PlatformToolset=v142 ..\deps\glfw\ALL_BUILD.vcxproj`
 - `copy /y ..\deps\glfw\src\Release\glfw3.dll .\`
-- `copy /y ..\deps\glfw\src\Release\glfw3dll.lib .\`
\ No newline at end of file
+- `copy /y ..\deps\glfw\src\Release\glfw3dll.lib .\`
+
+### WSL2
+
+Run `export GALLIUM_DRIVER=d3d12`
```

Markdownlint

Files should end with a single newline character

(MD047, single-trailing-newline)